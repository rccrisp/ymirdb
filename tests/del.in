set a 1 2 3
set b a 4 5 6
del a
del b
del a
list entries
bye