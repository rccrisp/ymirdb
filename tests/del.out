> ok

> ok

> not permitted

> ok

> ok

> no entries

> bye
